www/js/controller/report.address.controller.js

```diff
@@ -24,39 +22,19 @@ angular.module('vamosJuntas').controller('ReportAddressController', ['$scope','$
   };
 
   var reportANewRisk = function() {
-    var found = false;
-    if($scope.placeDetails.address === undefined){
-      $scope.placeDetails = addPlaceFromForm($scope.report);
-      $http.post(ApiEndpoint.url, $scope.placeDetails, {})
-      .success(function (data, status, headers, config) {
-        console.log('funcionou');
-      })
-      .error(function (data, status, header, config) {
-        console.log('não funcionou');
-      });
-    }else{
-      for(var i = 0; i < $scope.placeDetails.occurrences.length; i++ ){
-        if ($scope.placeDetails.occurrences[i].risk === $scope.report.risk) {
-          $scope.placeDetails.occurrences[i].reports.push({
-            "date": $scope.report.date,
-            "period": $scope.report.period
-          });
-          $scope.placeDetails.occurrences[i].count += 1;
-          found = true;
-          break;
-        }
-      }
-      if (!found) {
-        $scope.placeDetails.occurrences.push(addPlaceFromForm($scope.report));
-      }
-    }
```

Comment:
I liked it! I agree with you folks that this statement is just bringing complexity to this controller since we don't need to group the occurrences here.